modules/javafx.controls/src/main/java/javafx/scene/control/SkinBase.java

```diff
@@ -242,32 +242,32 @@ protected final void registerChangeListener(ObservableValue<?> property, Consume
     }
 
     /**
-     * Subclasses can invoke this method to register that they want to listen to
-     * invalidation events for the given observable. Registered {@link Consumer} instances
-     * will be executed in the order in which they are registered.
+     * Registers an operation to perform when the given {@code Observable} sends an invalidation event.
+     * Does nothing if observable or operation is {@code null}.
```

Comment:
I would write "Does nothing if either {@code observable} or {@code operation} are {@code null}"

Reply:
Done. Also changed leading upper-case of observable in first sentence to lower-case - for consistency because I don't see a difference between both, blind me? ;)